base_station/gui/src/components/AutonTask.vue

```diff
@@ -61,7 +59,7 @@ import OdometryReading from './OdometryReading.vue'
 import ArmControls from './ArmControls.vue'
 import DriveControls from './DriveControls.vue'
 import EncoderCounts from './EncoderCounts.vue'
-import WaypointEditor from './WaypointEditor_Auton.vue'
+import WaypointEditor from './WaypointEditor.vue'
 // import AutonJoystickReading from './AutonJoystickReading.vue'
```

Comment:
is this in use at all? If not, could we remove it

Reply:
Fixed